### DISCUSSION TITLE
点击重启docker，不显示成功，但是已经重启成功了

### DISCUSSION THREAD

**Discussion by he529406** [1Panel-dev/1Panel#5079] [Q&A]
### 联系方式

he529406@163.com

### 1Panel 版本

v1.10.7-lts

### 问题描述

docker重启后，显示成功重启

### 重现步骤

点击重启docker，不显示成功，但是已经重启成功了

**Comment by wanghe-fit2cloud** [comment]
1Panel 服务器是什么操作系统？
> 本地开发环境未能复现（Ubuntu 20.04、CentOS 7.9）
